package.json

```diff
@@ -95,7 +96,8 @@
     "webpack-assets-manifest": "5",
     "webpack-cli": "4",
     "webpack-dev-server": "^4.15.0",
-    "webpack-merge": "5"
+    "webpack-merge": "5",
+    "web3": "^1.10.0"
```

Comment:
I noticed in some cases we're version locking here and some we're not. Is there any distinguishing pattern for one is used over the other here?

Reply:
We should version lock whenever possible